src/main/java/christmas/view/InputView.java

```diff
@@ -0,0 +1,21 @@
+package christmas.view;
+
+import camp.nextstep.edu.missionutils.Console;
+import christmas.constant.Constants;
+
+public class InputView {
+
+    public String inputVisitDate() {
+        System.out.printf((Message.VISIT_DATE_REQUEST_MESSAGE.getMessage()) + "\n", Constants.MONTH);
```

Comment:
실행 환경에 따라 개행 문자가 달라질 수 있을 것 같습니다!
자바에서 각 상황에 따라 적절한 개행 문자를 반환해 주는 메서드를 제공해 주니 사용 해 보시는 것도 좋을 것 같습니다!
```suggestion
        System.out.printf((Message.VISIT_DATE_REQUEST_MESSAGE.getMessage()) + System.lineSeparator(), Constants.MONTH);
```